repos/spack_repo/builtin/packages/hip/package.py

```diff
@@ -462,16 +450,16 @@ def patch(self):
             )
         perl = self.spec["perl"].command
 
-        if self.spec.satisfies("@5.6:"):
+        if self.spec.satisfies("@5.7:"):
```

Comment:
This is not needed, if we don't support older versions anymore, right?

Reply:
It can be allowed as a reused spec, or as an external. In both cases we won't be able to "reinstall" it.